[CI] Failure in org.elasticsearch.indices.state.RareClusterStateIT.testDelayedMappingPropagationOnReplica

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+periodic/379/

Does not reproduce
```
./gradlew :server:integTest -Dtests.seed=F9699D9733C3983F \
  -Dtests.class=org.elasticsearch.indices.state.RareClusterStateIT \
  -Dtests.method="testDelayedMappingPropagationOnReplica" \
  -Dtests.security.manager=true -Dtests.locale=ar-TN -Dtests.timezone=Asia/Yakutsk \
  -Dcompiler.java=11 -Druntime.java=8
```
```
02:01:35   1> [2018-12-19T11:01:22,864][INFO ][o.e.i.s.RareClusterStateIT] [testDelayedMappingPropagationOnReplica] [RareClusterStateIT#testDelayedMappingPropagationOnReplica]: cleaned up after test
02:01:35   1> [2018-12-19T11:01:22,864][INFO ][o.e.i.s.RareClusterStateIT] [testDelayedMappingPropagationOnReplica] after test
02:01:35 FAILURE 29.9s J4 | RareClusterStateIT.testDelayedMappingPropagationOnReplica <<< FAILURES!
02:01:35    > Throwable #1: java.lang.AssertionError: 
02:01:35    > Expected: an instance of org.elasticsearch.action.index.IndexResponse
02:01:35    >      but: null
02:01:35    > 	at __randomizedtesting.SeedInfo.seed([F9699D9733C3983F:85175C586324A3DC]:0)
02:01:35    > 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
02:01:35    > 	at org.elasticsearch.indices.state.RareClusterStateIT.lambda$testDelayedMappingPropagationOnReplica$5(RareClusterStateIT.java:413)
02:01:35    > 	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:847)
02:01:35    > 	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:821)
02:01:35    > 	at org.elasticsearch.indices.state.RareClusterStateIT.testDelayedMappingPropagationOnReplica(RareClusterStateIT.java:409)
02:01:35    > 	at java.lang.Thread.run(Thread.java:748)
02:01:35    > 	Suppressed: java.lang.AssertionError: 
02:01:35    > Expected: an instance of org.elasticsearch.action.support.master.AcknowledgedResponse
02:01:35    >      but: null
```

Previous email threads suggest there's a connection to https://github.com/elastic/elasticsearch/issues/33673

I observed this failure on a Centos 7 VM after ~ 1 hour of running internal cluster tests in a loop. Stack trace is different not sure if it's related. I haven't experienced slowness on CentOS in CI so far.
```
2> REPRODUCE WITH: ./gradlew :server:integTest -Dtests.seed=BD345D2CF064C409 -Dtests.class=org.elasticsearch.indices.state.RareClusterStateIT -Dtests.method="testDelayedMappingPropagationOnReplica" -Dtests.security.manager=true -Dtests.locale=br-FR -Dtests.timezone=Asia/Jerusalem -Dcompiler.java=11 -Druntime.java=11
FAILURE 3.55s J2 | RareClusterStateIT.testDelayedMappingPropagationOnReplica <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: AcknowledgedResponse failed - not acked
   > Expected: <true>
   >      but: was <false>
   >    at __randomizedtesting.SeedInfo.seed([BD345D2CF064C409:C14A9CE3A083FFEA]:0)
   >    at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
   >    at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked(ElasticsearchAssertions.java:112)
   >    at org.elasticsearch.indices.state.RareClusterStateIT.testDelayedMappingPropagationOnReplica(RareClusterStateIT.java:340)
   >    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
   >    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
   >    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
   >    at java.base/java.lang.reflect.Method.invoke(Method.java:566)
   >    at java.base/java.lang.Thread.run(Thread.java:834)
```

--- a/server/src/test/java/org/elasticsearch/indices/state/RareClusterStateIT.java
+++ b/server/src/test/java/org/elasticsearch/indices/state/RareClusterStateIT.java
@@ -333,12 +333,12 @@ public class RareClusterStateIT extends ESIntegTestCase {
 
         // Force allocation of the primary on the master node by first only allocating on the master
         // and then allowing all nodes so that the replica gets allocated on the other node
-        assertAcked(prepareCreate("index").setSettings(Settings.builder()
+        prepareCreate("index").setSettings(Settings.builder()
                 .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
                 .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 1)
-                .put("index.routing.allocation.include._name", master)).get());
-        assertAcked(client().admin().indices().prepareUpdateSettings("index").setSettings(Settings.builder()
-                .put("index.routing.allocation.include._name", "")).get());
+                .put("index.routing.allocation.include._name", master)).get();
+        client().admin().indices().prepareUpdateSettings("index").setSettings(Settings.builder()
+                .put("index.routing.allocation.include._name", "")).get();
         ensureGreen();
 
         // Check routing tables

